resolwe/flow/executors/__init__.py

```diff
@@ -5,6 +5,8 @@
 ==============
 
 .. automodule:: resolwe.flow.executors.__main__
+.. automodule:: resolwe.flow.executors.run
```

Comment:
Could you refactor `resolwe.flow.executors.run`'s docstring so that the _Flow Executors_ title is not repeated twice?

Perhaps you could title it _Base Class_, like in _Flow Managers_ documentation?